### DISCUSSION TITLE
A mechanism to let end user redesign the keybinding system, would that be possible?

### DISCUSSION THREAD

**Discussion by glyh** [neovim/neovim#29068] [Ideas]
I want to utilize neovim's ecosystem, but I would like to try out kakoune's style binding, is that possible? 

If so, what should be changed by neovim? 

If I made an PR to neovim, would that be accepted?

Thanks a lot!

**Comment by clason** [comment]
Not really feasible, and such a PR would _not_ be welcome. You basically would have to rewrite the large swatches of the (Vim!) code from scratch, and likely break every plugin in existence. The common wisdom is "if you want kakoune bindings, just use kakoune" (you'll be back).
